app/assets/stylesheets/components/sub_nav.scss

```diff
@@ -20,6 +20,7 @@
       display: inline-block;
       text-decoration: none;
       border-bottom: 5px solid white;
+      font-size: 16px;
```

Comment:
I'd use a font size override class (`govuk-!-font-size-16`) rather than this custom CSS. Helps call out that we're doing something weird, easier to change in the future, etc.

https://design-system.service.gov.uk/styles/typography

Reply:
I feel this just clutters up the HTML, as well as I thought we try and avoid using this type of styling, as we use it as a last resort when styling? @kjdchapman

Reply:
Had a chat in-person about this. Think you're right that since there's already a custom class (with a bunch of padding/margins) here it's better to be consistent.